examples/actualComponents/Panel.js

```diff
@@ -97,15 +97,15 @@ class SwipeablePanel extends Component {
 
   componentDidMount = () => {
     this.animatedValueY = 0;
-    this.state.pan.y.addListener(value => (this.animatedValueY = value.value));
+    this.state.pan.y.addListener((value) => (this.animatedValueY = value.value));
 
     this.setState({ isActive: this.props.isActive });
   };
 
   componentDidUpdate(prevProps) {
     const { isActive, openLarge, onlyLarge } = this.props;
 
-    if (prevProps.isActive !== isActive) {
+    if (prevProps.isActive !== isActive || this.props.children !== prevProps.children) {
       this.setState({ isActive });
 
       if (isActive) {
```

Comment:
Just wondering if this could also cause issues down the way, should this not be a callback of `setState`?

Reply:
the rest of the code doesn't rely on sync set state. I think it will be fine but we will QA this for any unexpected behaviour